src/eos-updater.c

```diff
@@ -119,13 +122,18 @@ main (gint argc, gchar *argv[])
 {
   g_autoptr(GMainLoop) loop = NULL;
   g_autoptr(OstreeRepo) repo = NULL;
-  g_auto(EosUpdaterData) data = EOS_UPDATER_DATA_INIT;
+  g_auto(EosUpdaterData) data = EOS_UPDATER_DATA_CLEARED;
```

Comment:
It is just a better name and avoids a confusion like we now have with `G_VALUE_INIT` which actually does not initialize `GValue` - you have to do it with `g_value_init` afterwards.